WorkingGroups/UI/Agenda.md

```diff
@@ -0,0 +1,111 @@
+
+
+**Meeting:**
+1. Introduction to the UI working group
+2. Talk about deliverables i.e the TSC working group.
+3. Talk about the expected timeline.
+
+**Homework:**
+Look into the Rust community’s working group board. Take inspiration from other communities working groups. Put your findings under a Github issue.
+
+### :pushpin: Week 2  
+
+*Week 2, aims at beginning the implementation of the project i.e working group notice board. There are two parts here the content and the UI implementation of the design.*
+
+**Meeting** 
+
+ Content / Documentation- 
+1. Based on everyone’s findings, brainstorm on what written content that can go into the working group board. (Section wise.)
+2. Divide the content writing tasks into individual jobs and encourage members to take up responsibility.
+
+ UI- 
+1. Talk about noticeboard design and discuss how we can begin implementation.
+2. Divide UI tasks on a priority basis. And encourage people to take up the tasks.
+
+**Homework** 
+
+1. Begin writing content for the TSC working group board.
+2. Put the noticeboard design under an issue for reference.
+3. Begin implementation of UI w.r.t to the design and priority.
+
+
+### :pushpin:  Week 3
+
+*Week 3 has been designed to review the implementation completed so far. Talking about the blockers and next steps. At the end of week 3, our goal is to start finalizing the content and UI implementation.*
+
+
+**Meeting:** 
+
+Content/Documentation: 
+1. Talk about the content written so far.
+2. Discuss content that can be added and deducted.
+
+
+UI: 
+1. Evaluate the implementation progress and go over the required tasks before usability testing and public launch.
+2. Discuss blockers and next steps.
+3. Re-divide and re-prioritize the tasks if required and encourage people to take up the tasks.
+
+
+**Homework:**
+
+
+1. Start finalizing the content part by asking for reviews from members of the documentation working group.
+2. Start finalizing the implementation of the UI.
+
+
+### :pushpin: Week 4
+
+*In week 4, we want to finish the job and start usability testing so we can start getting the job done.*
+
+**Meeting:**
+
+
+Content/Documentation:
+1. Finalize the content under the respective sections. Talk about the missing pieces of the puzzle.
+
+
+UI:
+1. Brainstorm ways to conduct usability testing. 
+2. Talk about ways to improve. (Priority basis.)
+
+
+**Homework:** 
+
+1. Conclude document work.
+2. Talk about improvements under an issue and work on them.
+3. Begin publishing the content on the noticeboard and begin to finalize the work
+
+
+### :pushpin: Week 5
+
+*In week 5, we want to start discussing how to initiate a public launch. (Within the community itself if required)*
+
+**Meeting:**
+
+
+1. Review final implementation. 
+2. Initiate strategy for public launch.
+
+**Homework:**
+
+Finalise the TSC noticeboard And begin thinking about the first two FLINT-UI pages– (Point/RothC)
```

Comment:
Let's include the GCBM page too! I think the examples are already pretty good, but the GCBM will be a real challenge, because there is so much information to present.

Reply:
I haven't added that yet. I think we'll be over-committing ourselves. And maybe keeping the projects in rotation may help?